client/my-sites/email/titan-set-up-thank-you/index.tsx

```diff
@@ -66,125 +38,64 @@ const TitanSetUpThankYou = ( {
 		}
 	);
 
-	let nextSteps = [
+	const footerDetails = [
 		{
-			stepKey: 'titan_whats_next_view_inbox',
-			stepTitle: translate( 'Access your mailboxes' ),
-			stepDescription: translate( 'Access your email from anywhere with our webmail.' ),
-			stepCta: (
-				<FullWidthButton
-					href={ getTitanEmailUrl(
-						titanAppsUrlPrefix,
-						emailAddress,
-						false,
-						`${ window.location.protocol }//${ window.location.host }${ mailboxesPath }`
-					) }
-					primary
-					onClick={ () => {
-						recordEmailAppLaunchEvent( {
-							provider: 'titan',
-							app: 'webmail',
-							context: 'checkout-thank-you',
-						} );
-					} }
-				>
-					{ translate( 'Your Mailboxes' ) }
-				</FullWidthButton>
-			),
-		},
-		{
-			stepKey: 'titan_whats_next_get_mobile_app',
-			stepTitle: translate( 'Get mobile app' ),
-			stepDescription: translate(
-				"Access your email on the go with Titan's Android and iOS apps."
-			),
-			stepCta: (
-				<FullWidthButton
-					href={ titanControlPanelUrl }
-					target="_blank"
-					onClick={ () => {
-						recordEmailAppLaunchEvent( {
-							provider: 'titan',
-							app: 'app',
-							context: 'checkout-thank-you',
-						} );
-					} }
-				>
-					{ translate( 'Get app' ) }
-					<Gridicon className="titan-set-up-thank-you__icon-external" icon="external" />
-				</FullWidthButton>
+			key: 'footer-manage-email',
+			title: translate( 'Manage your email and site from anywhere' ),
+			description: translate(
+				'The Jetpack mobile app for iOS and Android makes managing your email, domain, and website even simpler.'
 			),
+			buttonText: translate( 'Get the app' ),
+			buttonHref: titanControlPanelUrl,
+			buttonOnClick: () => {
+				recordEmailAppLaunchEvent( {
+					provider: 'titan',
+					app: 'app',
+					context: 'checkout-thank-you',
+				} );
+			},
 		},
 		{
-			stepKey: 'titan_whats_next_manage_email',
-			stepTitle: translate( 'Manage your email' ),
-			stepDescription: translate(
-				'Add or delete mailboxes, migrate existing emails, configure a catch-all email, and much more.'
-			),
-			stepCta: (
-				<FullWidthButton href={ emailManagementPath }>{ translate( 'Manage' ) }</FullWidthButton>
+			key: 'footer-questions-email',
+			title: translate( 'Email questions? find answers' ),
```

Comment:
Should the `f` in `find answers` be capitalized? IMO, the phrasing here is a little weird. Because "find" is a verb, you kind of assume "email" would be a verb too, but here, it's referring to "questions about Email." Maybe it would be better if it wasn't a question, but something simpler like "Find answers to your questions about Email"

Reply:
Yeah, IMO your suggestion looks better 👍
@wongasy do you agree with the suggestion?

Reply:
@nightnei I think we should follow what was outlined in the copy document instead of figma. This is what we have in the doc:
> Email questions? We have the answers